src/plugins/saved_objects_management/public/management_section/objects_table/components/header.test.tsx

```diff
@@ -38,12 +38,38 @@ describe('Header', () => {
       onExportAll: () => {},
       onImport: () => {},
       onRefresh: () => {},
+      onCopy: () => {},
+      title: 'Saved Objects',
+      selectedCount: 0,
       totalCount: 4,
       filteredCount: 2,
+      workspaceEnabled: false,
+      hideImport: false,
     };
 
     const component = shallow(<Header {...props} />);
 
     expect(component).toMatchSnapshot();
   });
 });
+
+describe('Header - workspace enabled', () => {
+  it('should render normally when workspace enabled', () => {
```

Comment:
Nit: the description sounds too general, I may suggest this:
```suggestion
  it('should render `Duplicate All` button when workspace enabled', () => {
```

Reply:
description updated